Pylint: R1710

```bash
mmpose/apis/test.py:142:0: R1710: Either all return statements in a function should return an expression, or none of them should. (inconsistent-return-statements)
mmpose/datasets/datasets/mesh/mesh_mix_dataset.py:38:4: R1710: Either all return statements in a function should return an expression, or none of them should. (inconsistent-return-statements)
```

--- a/mmpose/apis/test.py
+++ b/mmpose/apis/test.py
@@ -183,3 +183,4 @@ def collect_results_gpu(result_part, size):
         # the dataloader may pad some samples
         ordered_results = ordered_results[:size]
         return ordered_results
+    return None

--- a/mmpose/datasets/datasets/mesh/mesh_mix_dataset.py
+++ b/mmpose/datasets/datasets/mesh/mesh_mix_dataset.py
@@ -45,3 +45,4 @@ class MeshMixDataset(Dataset, metaclass=ABCMeta):
                     self.datasets[i]) / self.length
                 index_new = int(np.round(index_new)) % (len(self.datasets[i]))
                 return self.datasets[i][index_new]
+        return None

